.gitlab-ci.yml

```diff
@@ -1,6 +1,9 @@
 # Stages for the CI/CD job
 stages:
   - setup
+  - lint
+  - format
+  - test
```

Comment:
Should `lint` and `format` really be independent stages / jobs? Considering this is CI/CD environment, and no changes to the source code are going to be committed, is there any reason to run the formatter at all? And doesn't it make sense to run the linter as part of the test job?